packages/o/openssl/makefile/patch.lua

```diff
@@ -0,0 +1,35 @@
+function _patch_for_llvm_rc(package, opt)
+    if package:is_plat("mingw") and package:has_tool("mrc", "llvm_rc", "llvm-rc", "rc") then
+        local cc = package:build_getenv("cc")
+        local cflags = opt.buildenvs.CFLAGS
+        local tmpfile = path.unix(os.tmpfile() .. ".c")
+        io.writefile(tmpfile, "int main(void) { return 0; }\n")
+        local compile_out, compile_err = try {function() return os.iorun(format("%s -v %s %s", cc, cflags, tmpfile)) end}
```

Comment:
use lua table as argv and call `os.iorunv(cc, argv)`

Reply:
I encountered an issue where the `CFLAGS` might be set to `-m64 --target=x86_64-w64-windows-gnu`, resulting in a concatenated command like `clang "-v" "-m64 --target=x86_64-w64-windows-gnu" a.c`. This causes the following error:

```
clang-19: error: unknown argument: '-m64 --target=x86_64-w64-windows-gnu'
clang version 19.1.6 (https://github.com/llvm/llvm-project.git e21dc4bd5474d04b8e62d7331362edcc5648d7e5)
Target: x86_64-w64-windows-gnu
Thread model: posix
InstalledDir: D:/Applications/Scoop/apps/mingw-mstorsjo-llvm-ucrt/19.1.6-20241217/bin
```

Reply:
because buildenvs.CFLAGS has been merged to string. not lua table.